DATABASE.md

```diff
@@ -0,0 +1,28 @@
+# Database Usage
+
+The MobileCoin wallet service stores its data in two places:
+- The first database is the ledger database, which is an append-only data store, recording the state of the public ledger. This contains no readable secret information, because all transaction outputs are stored on the ledger in an encrypted format. Its location is specified with the command line option `--ledger_db`.
```

Comment:
Nit: `--ledger-db`, similarly to `--wallet-db`

Reply:
Good catch.